pynta/utils.py

```diff
@@ -113,6 +114,11 @@ def get_unique_sym_struct_indices(geoms):
 
     for i,geom in enumerate(geoms_copy):
         adsorbate_atom_obj = geom
+        for constraint in adsorbate_atom_obj.constraints:
+            if isinstance(constraint, FixAtoms):
+                c = constraint # Capturing constraints of slab
+        adsorbate_atom_obj.set_constraint() # Reset constraints
+        adsorbate_atom_obj.set_constraint(c) # Apply slab constraints
```

Comment:
Why do we want the slab constraints to be applied?

Reply:
I don't know if we do want slab constraints applied. I just noticed that having slab constraints did not lead to a convergence error.
I can look into this further and see if we should just remove them all